docs/user_guide/trace.md

```diff
@@ -467,6 +467,32 @@ flag as follows:
 $ tritonserver --trace-config mode=opentelemetry \
     --trace-config opentelemetry,url=<endpoint> ...
 ```
+
+Triton's OpenTelemetry trace mode uses
+[Batch Span Processor](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#batch-span-processor),
+which batches ended spans and sends them in bulk. Batching helps
+with data compression and reduces the number of outgoing connections
+required to transmit the data. This processor supports both size and
+time based batching. Size-based batching is controlled by 2 parameters:
+`bsp_max_export_batch_size` and `bsp_max_queue_size`, while time-based batching
+is controlled by `bsp_schedule_delay`. Collected spans will be exported when
+the batch size reaches `bsp_max_export_batch_size`, or delay since last export
+reaches `bsp_schedule_delay`, whatever comes first. Additionally, user should
+make sure that `bsp_max_export_batch_size` is always less than
+`bsp_max_queue_size`, otherwise the excessive spans will be dropped
+and trace data will be lost.
+
+Default parameters for the Batch Span Processor are provided in
+[`OpenTelemetry trace APIs settings`](#opentelemetry-trace-apis-settings).
+As a general recommendation, make sure that `bsp_max_queue_size` is large enough
+to hold all collected spans, and `bsp_schedule_delay` does not cause frequent
+exports, which will affect Triton Server's latency. Also, users should keep
+in mind that minimal number of produced spans on Trton Server side is 3:
+The `InferRequest` span, which collects timestamps for when request was received
+by Triton, and when the request was sent; the request span, which is named
+after the model the request was sent to and records when request was started and
+ended; the `compute` span, which records compute timestamps.
```

Comment:
This section/description sounds a little unclear. Currently it says the 3 spans are:
1. `InferRequest`
2. request
3. `compute`

"InferRequest" and "request" sound a bit ambiguous/confusing to me.

(short term)

Without changing any current logic/names in the code, maybe calling the middle span the "Scheduler" or "Model" span would be more clear?:
1. `InferRequest`
2. Scheduler/Model
3. `Compute`

(longer term)

Or if we were actually changing some names in the codes/tests, then maybe something like this would eventually be clearest:
1. Network (HTTP/GRPC RECV/SEND)
2. InferRequest or Scheduler (REQUEST START/END)
3. Compute (COMPUTE START/END)

What do you think?

Reply:
I would like to avoid changing span names at least in this PR. I agree with the idea of calling the middle span as something related to Scheduler/Model. Let me think for a name

Reply:
I did my best in trying to clarify each type of spans